proposals/privacy.md

```diff
@@ -103,23 +102,28 @@ a particular person.  We will call these "privacy sensitive contexts".
 
 ### TDs MUST not contain PII.
 For example, a TD must not contain the email of a person with which it is associated or embed their name, address, or other PII 
-in a description or title field.
+in a description or title field, unless specifically required by the use cases. 
+ML: It must be under the descretion of the owner of the thing to chose to whom to expose which PII.
+There are cases where a TD without PII is useless.
 
 ### TDs MUST be transmitted only to Authenticated and Authorized consumers.
 Services providing TDs must require authentication (for example, using OAuth or PKI) of the requester to ensure they
 are who they say they are.  In addition, the system needs to ensure that the requester is authorized to receive TDs.
 
 ### Consumers MUST accept TDs only from Authenticated producers
-In other words, communication of TDs between producers and consumers must be over mutually authenticated channels.
+In other words, communication of TDs between producers and consumers must be over mutually authenticated channels,
+or the authenticity of a producer is validated by some other means (e.g. cryptographic signatures).
 
 ### TDs provided to a Consumer MUST be filtered to remove any optional information not required by that Consumer.
 For example, if a Consumer does not need human-readable titles or descriptions in a TD, these should be removed.
+ML: How does a producer know about the capabilities / requirements of a consumer? We don't have a mechanism to convey this information. In many scenarios a TD producer does not know about the potential consumers. This comment applies also o the points below.
```

Comment:
Part of this is to set up requirements for discovery.   When I do a query on a directory, I would request certain classes of information, and I would also have access rights for certain classes of information.   The Directory would then filter TDs to only give me the classes of data that I have asked for (this limits exposure and also improve efficiency) and have rights to.